apps/src/applab/designElements/label.jsx

```diff
@@ -275,18 +275,27 @@ export default {
           maxWidth: maxWidth + 'px'
         })
         .appendTo($(document.body));
-
```

Comment:
Nit: Was the whitespace removed for a reason?

Reply:
oops nope